project/src/main/java/edu/depaul/cdm/se452/project/model/ProfessorRepository.java

```diff
@@ -0,0 +1,15 @@
+package edu.depaul.cdm.se452.project.model;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+
+//repository class to help perform CRUD operations on the professors table
+public interface CourseRepository extends CrudRepository<Student, Long> {

```

Comment:
Interface name should be ProfessorRepository here and should extend CrudRepository<Professor, Long>